ibm/resource_ibm_is_snapshot.go

```diff
@@ -194,7 +197,54 @@ func resourceIBMISSnapshotCreate(d *schema.ResourceData, meta interface{}) error
 
 	snapshot, response, err := sess.CreateSnapshot(options)
 	if err != nil || snapshot == nil {
-		return fmt.Errorf("Error creating Snapshot %s\n%s", err, response)
+		if response.StatusCode == 409 {
+			restartError := "snapshots_attached_instance_not_running"
+			responseResultByte, errIns := json.Marshal(response)
+			if errIns == nil {
+				responseResultStr := string(responseResultByte)
+				if strings.Contains(responseResultStr, restartError) {
```

Comment:
Can we check if there are any other error messages associated with 409? if yes, is there any point in retrying to create snapshot?